conga-aem-definitions/src/main/roles/aem-dispatcher-cloud.yaml

```diff
@@ -353,9 +353,27 @@ config:
       xssProtection:
       # Enables/Configures the Cache-Control header on publish dispatcher (see https://developer.mozilla.org/en/docs/Web/HTTP/Headers/Cache-Control)
       cacheControl:
+        # default cache directive
+        - directives: "public, must-revalidate"
+          action: set
```

Comment:
we should remove "action: set" as this is the default? or add "action" everywhere here